Framework/PythonInterface/core/src/PythonStdoutChannel.cpp

```diff
@@ -0,0 +1,39 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#include "MantidPythonInterface/core/PythonStdoutChannel.h"
+#include <iostream> // streamsize
+
+#include <Python.h>
```

Comment:
The channel addition is very nice.

Just one small point for future reference. We can't use "Python.h" directly as we don't have a debug build of Python on Windows and it leads to errors like: `Cannot find library ...\python38_d.lib`

Everywhere we need `#include <Python.h>` we need to do instead do `#include "MantidPythonInterface/core/WrapPython.h"`